recipes/iiboost/build.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export ENABLE_OPENMP=no
+else
+    export ENABLE_OPENMP=yes
+fi
+
+mkdir build && cd build
+cmake \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DITK_DIR=$PREFIX/lib/cmake/ITK-4.11/ \
+    -DIIBOOST_USE_OPENMP=$ENABLE_OPENMP \
+	..
+
+make
```

Comment:
I just tried to set the default protocol to zero on top of the test files. Is there something else I should've done? Perhaps some conversion tool? (sorry I am not that well versed with the pickle intricacies)